apps/prairielearn/src/pages/instructorRequestCourse/instructorRequestCourse.html.ts

```diff
@@ -12,18 +15,33 @@ export const CourseRequestRowSchema = z.object({
 });
 type CourseRequestRow = z.infer<typeof CourseRequestRowSchema>;
 
+export const Lti13CourseRequestInputSchema = z
+  .object({
+    'cr-firstname': z.string(),
+    'cr-lastname': z.string(),
+    'cr-email': z.string(),
+    'cr-shortname': z.string(),
+    'cr-title': z.string(),
+    'cr-institution': z.string(),
+  })
+  .nullable();
+export type Lti13CourseRequestInput = z.infer<typeof Lti13CourseRequestInputSchema>;
+
 export function RequestCourse({
   rows,
+  lti13Info,
   resLocals,
 }: {
   rows: CourseRequestRow[];
+  lti13Info: Lti13CourseRequestInput;
   resLocals: Record<string, any>;
 }) {
   return html`
     <!doctype html>
     <html lang="en">
       <head>
         ${HeadContents({ resLocals, pageTitle: 'Request a Course' })}
+        ${compiledScriptTag('instructorRequestCourseLti13.ts')}
```

Comment:
If you're introducing an asset for this page, move the contents of the inline script tag there as well.

Reply:
Done in 0a22979fe. Updated them a bit too for deprecation and TS.